deltachat-ffi/deltachat.h

```diff
@@ -3721,6 +3709,20 @@ int             dc_chat_can_send              (const dc_chat_t* chat);
 int             dc_chat_is_protected         (const dc_chat_t* chat);
 
 
+/**
+ * Returns true if the chat was protected, and then an incoming message broke this protection.
+ *
+ * 1:1 chats are automatically set as protected when a contact is verified.
+ * When a message comes in that is not encrypted / signed correctly,
+ * the chat is automatically set as unprotected again.
+ * dc_chat_is_protection_broken() will return true until dc_accept_chat() is called.
```

Comment:
Fitting something about the feature flag into this sentence seemed to complicate it too much, so I added a note further up:

```
 * This function is only useful if the UI enabled the `verified_one_on_one_chats` feature flag,
 * otherwise it will return false for all chats.
```